README.md

```diff
@@ -37,11 +37,12 @@ We are currently building a team of community volunteers and educators to mainta
 
 There are many ways to contribute to Docs:
 
-- Submit a Pull Request to edit an existing entry.
-- Submit a Pull Request to create a new entry.
-- Submit a typo/bug in [GitHub Issues](https://github.com/Codecademy/docs/issues).
-- Reply to the thread on [Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868) to share ideas and collaborate with other contributors, maintainers, or super users to determine good future issues.
-- Share your thoughts in this [Docs feedback form](https://codecademyready.typeform.com/to/hzVIWDgz) to tell us how we can improve Docs!
+1. Get assigned to an Issue of your interest.
+   - Ask to be assigned to one of the [available Issues](https://github.com/Codecademy/docs/issues?q=is%3Aissue+is%3Aopen+-label%3Aclaimed).
+   - Submit a typo/bug in [GitHub Issues](https://github.com/Codecademy/docs/issues).
+2. Submit a Pull Request for your assigned Issue.
```

Comment:
Can we include a step explicitly mentioning that you should write an entry?

I.e.: 2. write your entry

Reply:
Use proper grammar of course.